units/systemd-vconsole-setup.service.in

```diff
@@ -9,11 +9,8 @@
 Description=Setup Virtual Console
 Documentation=man:systemd-vconsole-setup.service(8) man:vconsole.conf(5)
 DefaultDependencies=no
-Conflicts=shutdown.target
-Before=sysinit.target shutdown.target
 ConditionPathExists=/dev/tty0
```

Comment:
Actually maybe it's too lax with those dependencies, even if the service is now de-facto a simple wrapper around systemd-vconsole-setup. Someone might decide to use it as a dependency for some stuff.